src/ArcxAnalyticsSdk.ts

```diff
@@ -391,6 +391,16 @@ export class ArcxAnalyticsSdk {
   /** Initialises the Analytics SDK with desired configuration. */
   static async init(apiKey: string, config?: Partial<SdkConfig>): Promise<ArcxAnalyticsSdk> {
     const sdkConfig = { ...DEFAULT_SDK_CONFIG, ...config }
+    /*
+      TODO: Remove this if statement aproximately in February.
+      Clean local storage in case if users with previous versions saved wrong indentity_id when ARCX_ANALYTIC_API_KEY was incorrect
+    */
+    if (
+      window.localStorage.getItem(IDENTITY_KEY)?.length !== 64 &&
+      window.localStorage.getItem(IDENTITY_KEY)?.length !== 0
```

Comment:
Can we remove second statement? If IDENTITY_KEY is found without being 64 characters, then doesn't that mean it shouldn't exist, so we always remove it? 

I guess I can't see a condition where `getItem(..)` is defined and not zero? But maybe I'm missing something

Reply:
I think you have the point, so I'll remove it.